framework/src/service/ServiceReferenceBasePrivate.cpp

```diff
@@ -64,9 +67,9 @@ InterfaceMapConstPtr ServiceReferenceBasePrivate::GetServiceFromFactory(
                                                     ServiceRegistrationBase(registration));
     if (!smap || smap->empty())
     {
-      registration->bundle->coreCtx->listeners.SendFrameworkEvent(FrameworkEvent(FrameworkEvent::Type::FRAMEWORK_WARNING, 
-                                                                                  MakeBundle(bundle->shared_from_this()), 
-                                                                                  std::string("ServiceFactory returned an empty or nullptr interface map."), 
+      registration->bundle->coreCtx->listeners.SendFrameworkEvent(FrameworkEvent(FrameworkEvent::Type::FRAMEWORK_WARNING,
```

Comment:
Actually, I would like to keep it as it is for now, even though it is slightly inconsistent. I would have to rewrite the whole doc block, just to change it back soon after when working on #217.